content/agent/basic_agent_usage/windows.md

```diff
@@ -46,6 +46,8 @@ The following configuration command line options are available when installing t
 | `PROXY_USER`      | String | If using a proxy, sets your proxy user. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                              |
 | `PROXY_PASSWORD`  | String | If using a proxy, sets your proxy password. [Learn more on using a proxy with the Datadog Agent][8].                                                                                                          |
 
+Note: If a valid `datadog.yaml` is found, that file takes precedence over all specified command-line options.
+
```

Comment:
If a valid `datadog.yaml` is found, and has an API_KEY configured....

For better or worse, the conditional is whether an API key is set.

Reply:
This was not the case when I tested it.... if the file was there, with no `API_KEY` the proxy settings did not get applied.